cfg.yaml

```diff
@@ -1,4 +1,12 @@
 ---
+- name: settings
```

Comment:
Please change the policies to be part of both the JIRA and EMAIL sections, and not a section by itself.

Reply:
I saw you added a dedicated "settings" section. I would like to have the policies inside the different integrations (JIRA, EMAIL) since this will allow us to have different policies for the different integrations.

Reply:
@jerbia ok. sure

Reply:
@jerbia maybe Should I add global settings and local settings?
and calculate the effective settings?